app/config/config.yml.dist

```diff
@@ -240,6 +247,9 @@ wysiwyg:
 #  encryption: null
 #  auth_mode: null
 
+#mailoptions:
+#  transport: mail
+
```

Comment:
If they uncomment both, the YML breaks, and an error is shown. If you combine them, i'm not sure it would be easier to grok for people. Neither is really very good.. Hmm..